common/src/main/groovy/ru/sberbank/school/task03/Main.java

```diff
@@ -0,0 +1,22 @@
+package ru.sberbank.school.task03;
+
+import ru.sberbank.school.task02.util.FxResponse;
+import ru.sberbank.school.task03.util.Formatter;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+
+        List<FxResponse> responses = Arrays.asList(
+            new FxResponse("USD/RUB", "100.00","200","16.06.19", "SELL", false),
```

Comment:
&#39;new&#39; has incorrect indentation level 12, expected level should be 16.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-9490556>*